src/ui/components/search/searchcomponent.js

```diff
@@ -141,6 +141,10 @@ export default class SearchComponent extends Component {
       this.debouncedSearch(q);
     });
 
+    this.core.globalStorage.on('update', StorageKeys.API_CONTEXT, () => {
+      this.debouncedSearch(this.query);
+    });
```

Comment:
Was this functionality referenced in the spec somewhere?

Reply:
This was not referenced in the spec. Max gave feedback that we would like to have the context persisted as we navigate /within/ the answers experience. That is, it should be reflected in the URLs. This search was the solution I landed on for updating the Nav tab links and the View more links with the updated context params. Do you have any thoughts?

Reply:
I think we should edit those bits of code making the nav tab urls and view more links directly. We can just pull whatever is in globalStorage for the context there. Doing things in an indirect way, like this, may have potential unwanted side-effects.

Reply:
Updated!